### PR TITLE
docs(s): updated all S components docs

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Documentation

#### What is expected?

Updated documentation of:

- ScrollView
- Select
- Separator
- Slider
- Sphere
- StateBar
- StatusIndicator
- StealthCopiable
- Stepper
- Switch
- SwitchButton

### CODE REVIEW

File: src/components/Slider/index.js [hunk 1]
```diff
@@ -158,14 +158,16 @@ const Slider = ({ children, ...props }) => {
   )
 }
 
-Slider.Item = ({ as, ...props }) => (
+export const Item = ({ as, ...props }) => (
```
Comment (DorianMaliszewski) [inline src/components/Slider/index.js:161]: const Item = ....

Slide.Item = Item ?

File: src/components/StateBar/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Box from '../Box'
 import ProgressBar from '../ProgressBar'
 import Typography from '../Typography'
 
-const State = ({ label, children, ...props }) => (
+export const State = ({ label, children, ...props }) => (
```
Comment (DorianMaliszewski) [inline src/components/StateBar/index.js:8]: Same const State = ....
StateBar.State = State

File: src/components/StateBar/index.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ const line = css`
   margin-top: 12px;
 `
 
-const Bar = ({ unlimited, value, ...props }) => {
+export const Bar = ({ unlimited, value, ...props }) => {
```
Comment (DorianMaliszewski) [inline src/components/StateBar/index.js:41]: Same

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
Awaiting for your responses

File: src/components/Slider/index.js (hunk 1 above)
Comment (matthprost) [reply src/components/Slider/index.js:161]: @DorianMaliszewski isn't it what I'm already doing ?

```js
export const Item = ({ as, ...props }) => (
  <StyledBorderWrapper as={as} {...props} draggable="true" />
)

Slider.Item = Item
```

**Review by DorianMaliszewski — APPROVED** [review]

**Review by chambo-e — APPROVED** [review]

**Pull request merged**